pyproject.toml

```diff
@@ -26,7 +26,6 @@ dependencies = [
     "torch-optimizer==0.3.0",
     "transforms3d==0.3.1",
     "typing==3.7.4.3",
-    "vtk>=9.2.6",
```

Comment:
What happened to vtk in the toml?

Reply:
Replaced it with the vtk install in Dockerfile (similar to Core one.) VTK is not shipping arm wheel yet, so` pip install nvidia-modulus.sym` fails in an arm machine due to this line.

Reply:
I understand this is needed to build on arm, but does this mean pip installs outside of a container will now break because of missing dependencies?

Reply:
That is a good suggestion indeed! After Nick's and Mohammad's comments, I realized that vtk is very central to Modulus Sym due to its inferencers/validators. Let me try this and get back